docs/sphinx/manual/creating_hpx_projects.rst

```diff
@@ -297,6 +297,28 @@ variable to point to the installation of |hpx|.
    in your targets. The |hpx| include directories are available with the
    ``HPX_INCLUDE_DIRS`` CMake variable.
 
+.. _hpxcxx_documentation:
+
+Building |hpx| Components and Applications
+------------------------------------------
+The ``hpxcxx.in`` compiler wrapper in ``hpx/cmake/templates/`` directory helps to build a |hpx| component, an application or generate a ``<file-name>.o`` file, based on the arguments passed to it.
```

Comment:
```suggestion
------------------------------------------

The ``hpxcxx`` compiler wrapper helps to build a |hpx| component, an application or generate a ``<file-name>.o`` file, based on the arguments passed to it.
```

Reply:
Please wrap the text at 80 characters.

Reply:
I'd simply leave out the location of the wrapper as it's meant to be installed so that it's available in `PATH`.

Reply:
Right. Got it. Thanks for the clarification :)

Reply:
Got it. Thanks for the corrections :)